src/main/scala/edu/berkeley/ce/rockslicing/LoadBalancer.scala

```diff
@@ -0,0 +1,323 @@
+package edu.berkeley.ce.rockslicing
+
+import breeze.linalg
+import breeze.linalg.{DenseVector, DenseMatrix}
+import scala.annotation.tailrec
+
+/** 
+  * Manages initial partitioning of rock volume to maintain load balance among partitions
+  */
+object LoadBalancer {
+
+  /**
+    * Produce processor joints that cut initial rock volume into approximately equal volumes.
+    * These joints will be used to seed the initial Block RDD such that the volume or rock
+    * provided to each process is approximately equal.
+    * @param rockVolume Initial block defining the entire rock volume of interest
+    * @param numSeedJoints Number of joints to process before initiating parallel block cutting.
+    *                      Should be one less than number of cores used.
+    * @return Set of processor joints that will be used to seed the initial RDD
+    */
+  def generateSeedJoints(rockVolume: Block, numSeedJoints: Integer):
+                         Seq[Joint] = {
+    // Calculate bounding box
+    val vertices = rockVolume.findVertices.values.flatten
+    val x_max = (vertices.map{ case vertex => vertex._1}).max
+    val x_min = (vertices.map{ case vertex => vertex._1}).min
+    val y_max = (vertices.map{ case vertex => vertex._2}).max
+    val y_min = (vertices.map{ case vertex => vertex._2}).min
+    val z_max = (vertices.map{ case vertex => vertex._3}).max
+    val z_min = (vertices.map{ case vertex => vertex._3}).min
+    val boundingBox = (x_min, y_min, z_min, x_max, y_max, z_max)
```

Comment:
This is getting picky, but I wonder if a bit more structure is in order here, i.e. defining `boundingBox` as a pair of 3-tuples, one for the minima and another for the maxima, or as a triple of pairs for the x, y, and z minimum and maximum.